app/views/atransactions/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= stylesheet_link_tag "transactions_new" %>
+
+<%= render "shared/navbar", category: params[:category_id] %>
+
+<section class="new-transaction-section">
+<%= render "form", atransaction: @atransaction %>
+
+  <br>
+
+  <div>
+    <%# <%= link_to "Back to transactions", category_path(@category) %>
```

Comment:
### Add a new transaction page
- [ ] According to the Entity relational database Diagram, there is a many-to-many relationship between Group and Entity so, It's one of the project requirements that The user fills out a form to create a new transaction or expense with these mandatory parameters:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one).  kindly prevent the user to create a new transaction without selecting at least one  categories 

![image](https://github.com/blueberry1312/rails-capstone/assets/79058364/62cdca53-5f6e-41bf-8b8f-8fbb9709a773)

- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻